src/net/sourceforge/kolmafia/session/ChoiceManager.java

```diff
@@ -2137,6 +2137,28 @@ public static void postChoice0(final String urlString, final GenericRequest requ
     ChoiceControl.postChoice0(choice, urlString, request);
   }
 
+  /**
+   * Determine if a request to choice.php showed that we weren't actually in a choice adventure
+   *
+   * @param request Completed request to check
+   * @return Whether KoL indicated that the player was not actually in a choice adventure
+   */
+  public static boolean bogusChoice(final GenericRequest request) {
+    if (!ChoiceManager.handlingChoice
+        || !request.getURLString().startsWith("choice.php")
```

Comment:
I'm defensively coding here against someone using this function unexpectedly in the future

Reply:
Fair enough.

Note that where this is called, GenericRequest has already extracted the urlString out of the request, so you could just pass it in, as we do for ```public static void postChoice0(final String urlString, final GenericRequest request)``` et. al. but I'm not going to require that.